app/ModelFunctions/AlbumFunctions.php

```diff
@@ -136,7 +136,9 @@ public function gen_thumbs($return, $album_list)
 		$photos = Photo::whereIn('album_id', $album_list)
 			->orderBy('star', 'DESC')
 			->orderBy(Configs::get_value('sorting_Photos_col'), Configs::get_value('sorting_Photos_order'))
-			->limit(3)->get();
+			->limit(3)
+			->get()
+			->sortBy(Configs::get_value('sorting_Photos_col'), SORT_NATURAL, (Configs::get_value('sorting_Photos_order') === 'DESC'));
```

Comment:
I did not test photos, and the sortBy must be after the ->get() so it will not work well in this situation. I think the sortBy should be removed here. I  will modify the code.